src/clj_sqs_extended/aws/sqs.clj

```diff
@@ -104,18 +107,24 @@
     message    - The actual message/data to be sent
 
   Options:
-    format     - The format (currently :json or :transit) to serialize outgoing
-                 messages with (optional, default: :transit)"
+    format     - The format (currently :json or :transit) to serialize the outgoing
+                 message with (optional, default: :transit)"
```

Comment:
it's in the `Options` section already, no need to say that it's `optional` again